index.md

```diff
@@ -24,7 +25,7 @@ The system either uses the contents of the old system, or the new image; Never s
 Updates can be delayed or scheduled.
 This default behavior can be adapted or controlled by a larger management system.
 
-1. If an update does not function correctly it is possible to roll back to the container image previously functioning before the update.
+1. If an update does not function correctly it is possible to roll back to the container image previously functioning before the update, or to any previously bootable version in the registry. 
```

Comment:
@castrojo Is this really a roll back though? @cgwalters Can we roll back to another version in the registry, or only what's on the system?

Reply:
We're doing this in practice with our images but now that you bring it up I'm not sure how officially supported that is. 

The use case is usually when people don't have a pinned snapshot or didn't realize they had a regression on something. The it's useful to be able to grab a remote older version to try to pin down where the regression happened.

Reply:
`switch` and `rollback` I think are 99% equivalent for this. (The difference is if there's any local changes in `/etc` that were made between the operations)